packages/overlay/src/integrations/sentry/types.ts

```diff
@@ -47,9 +47,9 @@ export type Breadcrumb = {
 
 type CommonEventAttrs = {
   // not always present, but we are forcing it in EventCache
+  type?: 'error' | 'event' | 'message' | 'default';
```

Comment:
hi @markushi  any specific reason for moving this to `commonEventAttrs`?

Reply:
either we should add "transaction" here as well and remove the type from SentryEvent as well.

OR

lets revert this and move these types to SentryErrorEvent again.